srcpkgs/goldendict-ng/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'goldendict-ng'
+pkgname=goldendict-ng
+version=24.05.05
+revision=1
+archs="x86_64"
+build_style=cmake
+configure_args="-DUSE_ALTERNATIVE_NAME=ON"
+makedepends="qt6-speech-devel qt6-svg-devel qt6-qt5compat-devel qt6-base-devel qt6-webengine-devel qt6-multimedia-devel qt6-speech-plugin-flite qt6-speech-plugin-mock qt6-speech-plugin-speechd pkg-config ffmpeg-devel hunspell-devel opencc-devel libvorbis-devel xapian-core-devel libeb-devel libzim-devel qt6-tools-devel"
```

Comment:
wrap at 80 chars (and indent following lines with a single space)

Reply:
Done. But why 80 chars? Most if not all screens today have support for much more.

Reply:
because that's the convention we follow:

> Most of the lines should be kept under 80 columns

https://github.com/void-linux/void-packages/blob/master/Manual.md#introduction